stripe-ui-core/src/main/java/com/stripe/android/uicore/image/StripeImage.kt

```diff
@@ -0,0 +1,77 @@
+package com.stripe.android.uicore.image
+
+import androidx.annotation.RestrictTo
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.BoxWithConstraints
+import androidx.compose.foundation.layout.BoxWithConstraintsScope
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.compose.ui.graphics.painter.BitmapPainter
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.unit.Dp.Companion.Infinity
+import androidx.compose.ui.unit.IntSize.Companion.Zero
+import kotlinx.coroutines.launch
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+@Composable
+fun StripeImage(
+    url: String,
+    placeholder: Painter,
+    imageLoader: StripeImageLoader,
+    contentDescription: String?,
+    modifier: Modifier = Modifier
+) {
+    BoxWithConstraints(modifier) {
+        val (width, height) = calculateBoxSize()
+        val painter: MutableState<Painter> =
+            remember { mutableStateOf(BitmapPainter(ImageBitmap(width, height))) }
```

Comment:
Also not in a style guide, but more concise:
```suggestion
        val painter by remember { mutableStateOf(BitmapPainter(ImageBitmap(width, height))) }
```
Then instead of getting/setting `painter.value`, just read/write to `painter`.
Could leave the type too, in this case it's clear from the variable name.

Reply:
https://github.com/stripe/stripe-android/pull/5307/commits/05fe92e9dfafd84d16bce6949f7d501f0a4b7bce